### ISSUE TITLE
WCAG 1.3.1: Ensures elements with an ARIA role that require child roles contain them (.ms-Pivot)

### ISSUE THREAD

**Issue by saychiraj** [microsoft/fluentui#27052]
#### Issue
Pivot component's overflow button is violating accessibility rule because it does not have role of "tab".
Pivot has a role "tablist" and all it's children are expected have a role of "tab"

`Ensures elements with an ARIA role that require child roles contain them` ([aria-required-children](https://accessibilityinsights.io/info-examples/web/aria-required-children))
#### Target application
[Templates | Power Pages](https://make.local.powerpages.microsoft.com/environments/b016e9b2-5b14-e167-bfcf-099bea4c55d1/portals/templateshub)
#### Element path
.ms-Pivot
#### Snippet
`<div role="tablist" aria-label="All templates" class="ms-FocusZone css-307 ms-Pivot root-559" data-focuszone-id="FocusZone658">`
#### How to fix
    Fix any of the following:
      Element has children which are not allowed (see related nodes)
      Element has no aria-busy="true" attribute
#### Environment
Microsoft Edge version 110.0.1587.46
This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by rogerprz** [comment]
Any updates on when this will be fixed?

**Comment by usmanhf** [comment]
This is a known issue. I found this in internal documentation:
"`aria-required-children` failure on Pivot with overflow
This bug exists because the ARIA tablist pattern does not currently support having an overflow menu (it's a known problem, and being worked on). Because we have an overflow menu, we have two options to implement it:

- The menu trigger is a menubutton, which is technically not an allowed child of a tablist (though negative user impact here is very minimal, since the menubutton triggers the correct mode in windows screen readers, similar to tabs).

- The menu trigger is a tab with aria-haspopup, which is not an allowed attribute on that role. Also not much negative user impact here.

Neither choice is perfect; we chose the first for v8 and the second would also be fine, though both would give you automated errors. Until this pattern becomes supported in the spec, those errors should be ignored."
